src/script/run-make.sh

```diff
@@ -54,6 +54,16 @@ function prepare() {
         which_pkg="debianutils"
     fi
 
+    if in_jenkins; then
+        if ! type clang-16 > /dev/null 2>&1 ; then
+            ci_debug "Getting clang-16"
+            wget https://apt.llvm.org/llvm.sh
+            chmod +x llvm.sh
+            $DRY_RUN sudo ./llvm.sh 16
```

Comment:
what the?......

clang doesn't even *package* the compiler?  This is insanity for a shared build host

Reply:
ubuntu 24.04 has a [clang-16](https://packages.ubuntu.com/noble/clang-16) package, but the jenkins builders are still running 22.04 which only goes up to [clang-15](https://packages.ubuntu.com/jammy/clang-15)

Reply:
It does.  The script adds a repository hosted at http://apt.llvm.org.

Reply:
@idryomov well that's better; thanks for the explanation